config/initializers/stripe.rb

```diff
@@ -1 +1,14 @@
-Stripe.api_key = ENV['STRIPE_SECRET_KEY']
\ No newline at end of file
+Stripe.api_key = ENV['STRIPE_SECRET_KEY']
+StripeEvent.configure do |events|
+  events.subscribe 'charge.succeeded' do |event|
+    user = User.where(customer_token: event.data.object.customer).first
```

Comment:
Done.  I totally agree with you, I was just following along with the solution on this part.